aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -156,6 +156,21 @@ Tensor& addr_out(Tensor &result, const Tensor& self, const Tensor& vec1, const T
   return at::_addr_out(result, self, vec1, vec2, beta, alpha);
 }
 
+Tensor& ger_out(Tensor &result, const Tensor& self, const Tensor& vec2) {
+  check_1d(self, "self", "ger");
+  check_1d(vec2, "vec2", "ger");
+  if (result.dim() != 2 || result.size(0) != self.size(0) || result.size(1) != vec2.size(0)) {
+    result.resize_({ self.size(0), vec2.size(0) });
+  }
+  return at::_addr_out(result, result, self, vec2, Scalar(0), Scalar(1));
+}
+
+Tensor ger(const Tensor& self, const Tensor& vec2) {
+  Tensor result = at::empty({0}, self.options());
+  at::native::ger_out(result, self, vec2);
```

Comment:
In general it's good to call `at::ger_out` instead of `at::native`. It doesn't matter in the current implementation, but it might cause trouble for XLA if we change `ger_out` to be non-compound op in the future.

Reply:
changed to at::